README.md

```diff
@@ -233,7 +233,13 @@ const openpgp = require('openpgp'); // use as CommonJS, AMD, ES6 module or via w
         decryptionKeys: privateKey
     });
     console.log(decrypted); // 'Hello, World!'
-    console.log(signatures[0].valid) // signature validity (signed messages only)
+    // check signature validity (signed messages only)
+    try {
+        await signatures[0].verified; // throws on invalid signature
+        console.log('signature is valid');
```

Comment:
Very nitpicky but, let's try to make the capitalization vaguely consistent (also below)
```suggestion
        console.log('Signature is valid');
```